include/uapi/sound/sof/abi.h

```diff
@@ -26,7 +26,7 @@
 
 /* SOF ABI version major, minor and patch numbers */
 #define SOF_ABI_MAJOR 3
-#define SOF_ABI_MINOR 6
+#define SOF_ABI_MINOR 7
```

Comment:
Since ABI MINOR 7 is used for trace time stamp already https://github.com/thesofproject/sof/commit/de5bba49aaf2c0060a279de09580ac4205f550cb#diff-743f02a545ca1c6532a4654cdc8f9cc4, should it be 8 instead? Same on the FW side https://github.com/thesofproject/sof/pull/1290/files#diff-743f02a545ca1c6532a4654cdc8f9cc4R55

Reply:
sorry, I was confused, this should be MINOR 10 according to Liam's list https://github.com/orgs/thesofproject/projects/2#card-20443578

Reply:
@ranj063 @xiulipan as requested I have merged FW part https://github.com/thesofproject/sof/pull/1290 which will use MINOR 8 (sharing the MINOR with link DMA updates). Dashboard updated.  https://github.com/thesofproject/sof/pull/1290 does not update FW ABI minor as the feature is not advertised until link DMA (with ABI bump) is merged.

Reply:
@lgirdwood I am confused now too. The FW patch uses ABI minor 7. Why should the driver use 8?